src/ui/screens/onboarding/OnboardingDevicesSlide.qml

```diff
@@ -0,0 +1,208 @@
+
+import QtQuick 2.5
+import QtQuick.Layouts 1.15
+
+import Mozilla.Shared 1.0
+import components 0.1
+import compat 0.1
+
+ColumnLayout {
+    id: root
+
+    signal nextClicked()
+    signal backClicked()
+
+    spacing: 0
+
+    MZHeadline {
+        Layout.topMargin: MZTheme.theme.windowMargin * 1.5
+        Layout.leftMargin: MZTheme.theme.windowMargin * 2
+        Layout.rightMargin: MZTheme.theme.windowMargin * 2
+
+        Layout.fillWidth: true
+
+        text: MZI18n.OnboardingDevicesSlideHeader
+        horizontalAlignment: Text.AlignLeft
+    }
+
+    MZInterLabel {
+        Layout.topMargin: 8
+        Layout.leftMargin: MZTheme.theme.windowMargin * 2
+        Layout.rightMargin: MZTheme.theme.windowMargin * 2
+        Layout.fillWidth: true
+
+        text: MZI18n.OnboardingDevicesSlideBody
+        horizontalAlignment: Text.AlignLeft
+        color: MZTheme.theme.fontColor
+    }
+
+    RowLayout {
+        Layout.topMargin: 25
+        Layout.leftMargin: MZTheme.theme.windowMargin * 2
+        Layout.rightMargin: MZTheme.theme.windowMargin * 2
+
+        MZBoldInterLabel {
+            Layout.alignment: Qt.AlignVCenter
+
+            text: MZI18n.OnboardingDevicesSlideDeviceTypeLabel
+            font.pixelSize: MZTheme.theme.fontSize
+            lineHeight: MZTheme.theme.labelLineHeight
+            verticalAlignment: Text.AlignVCenter
+        }
+
+        Item {
+            Layout.fillWidth: true
+        }
+
+        MZSegmentedToggle {
+            id: deviceTypeToggle
+
+            Layout.preferredWidth: 108
+
+            model: ListModel {
+                ListElement {
+                    segmentIconPath: "qrc:/ui/resources/android.svg"
+                    //Android icon is too detailed for how small we are presenting it here (24x24) causing it to appear poorly/grainy when behind a ColorOverlay
+                    //so we use an alternate svg with the color built in
+                    selectedSegmentIconPath: "qrc:/ui/resources/android-active.svg"
+                    segmentLabelStringId: "OnboardingDevicesSlideDeviceTypeAndroid"
+                    segmentButtonId: "tabAndroid"
+                }
+                ListElement {
+                    segmentIconPath: "qrc:/ui/resources/apple.svg"
+                    //When ColorOverlay is applied to the apple icon, it "thickens" a little bit which is a somewhat jarring when the transition from unselected -> selected happens
+                    //so we use an alternate svg with the color built in
+                    selectedSegmentIconPath: "qrc:/ui/resources/apple-active.svg"
+                    segmentLabelStringId: "OnboardingDevicesSlideDeviceTypeApple"
+                    segmentButtonId: "tabApple"
+                }
+            }
+
+            Component.onCompleted: {
+                switch(Qt.platform.os) {
+                    case "osx":
+                    case "ios":
+                        selectedIndex = 1
+                        break
```

Comment:
Should `iPadOS` fall under the same SelectedIndex?

Reply:
unfortunately, Qt does not differentiate between iOS and iPadOS - [source](https://doc.qt.io/qt-6/qml-qtqml-qt.html#platform-prop)

But it's about time that they should!